### ISSUE TITLE
Does not work with Mail.app Version 9

### ISSUE THREAD

**Issue by willtmc** [nompute/GMailinator#40]
This plugin does not work with the latest version of Mail, Version 9. I don't know how to change it to make it work but I can share error messages if it would be helpful.

**Comment by eibrahim** [comment]
+1

**Comment by RaVbaker** [comment]
To solve this issue please follow this procedure on your computer: https://github.com/nompute/GMailinator/issues/22#issuecomment-59677606 It worked for me. :) 

Only difference was location of `/Library/Mail`. On my machine I had to put `~` in front of this path, like: `~/Library/Mail/Bundles/GMailinator.mailbundle/Contents/Info.plist` and the Bundles folder was named like: "Bundles (disabled)" so I changed Info.plist and moved it back to Bundles

**Comment by jacob-ogre** [comment]
Yep, #22 also worked for me.  Thanks for the bundle, @nompute, and the solution, @RaVbaker.

**Comment by serialmetrics** [comment]
Does anyone have the updated version for Mac OS Sierra?

**Comment by serialmetrics** [comment]
Any updates for Sierra, specifically Mail 10.2
